torch_xla/csrc/convolution.cpp

```diff
@@ -316,6 +320,533 @@ ConvGrads BuildTransposedConvolutionBackward(
 
 }  // namespace
 
+std::string PTXLAToString(PTXLATensorFormat format) {
```

Comment:
for any code that is copy/inspired by TF's implementation, please leave a comment in the top of the function to point to the TF code. This way
1. people know this logic is from TF
2. if TF's implementation ever change we might also want to track it

Reply:
Thanks, Jack, updated